Tests/SolanaTests/AsyncAssertThrowing.swift

```diff
@@ -1,6 +1,7 @@
 import Foundation
 import XCTest
 
+@available(iOS 13.0.0, *)
```

Comment:
I think this @available is because async/await is not always available, right? If so, it's actually the language version you want to check, not the OS version. Theoretically, this library could run on macOS, linux, or other platforms too, so you should probably use `@available(swift 5.5)` (I think 5.5. is the minimum swift version that supports async)

Reply:
This one is annotation what Xcode put there as fix after complaining.